cluster-autoscaler/FAQ.md

```diff
@@ -88,7 +88,12 @@ Cluster Autoscaler decreases the size of the cluster when some nodes are consist
   * are not run on the node by default, *
   * don't have a [pod disruption budget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#how-disruption-budgets-work) set or their PDB is too restrictive (since CA 0.6).
 * Pods that are not backed by a controller object (so not created by deployment, replica set, job, stateful set etc). *
-* Pods with local storage. *
+* Pods with local storage. *  
+    - unless the pod has the following annotation set:
```

Comment:
I took the liberty of adding a sub-bullet here. Happy to change if you have any feedback.

Reply:
The preview is a little messed up. Here's what it actually looks like:
![image](https://user-images.githubusercontent.com/34534103/230542802-b4d43105-e268-4cb2-9a4d-9d765a23ae04.png)
https://github.com/kubernetes/autoscaler/blob/df1b7e73e73ac619c83623588421096b3f19ddb8/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node

Reply:
The formatting looks good.

nit: I'd change the wording from "where..." to "and all of the pod's local volumes are listed in the annotation value", it's a bit more accurate.

Reply:
I think it makes things clearer too. Thank you for the suggestion. Will update the PR.